docs/sql-guide/functions/function-charindex.md

```diff
@@ -7,7 +7,7 @@ grand_parent: SQL guide
 
 # CHARINDEX() function
 
-The `Charindex()` function supplies an integer value that represents the start of a substring within a string value
+The `Charindex()` function supplies an integer value that represents the start of a substring within a string value. Returns -1 if the substring cannot be found.
```

Comment:
Returns -1 should be added as an additional row to the "Returns" table for the same reason as ASCII()

For example

```
| Data type | Value |
|---|---|
| `int` | Position of `substring` in the `expr` input string. |
| `int` | -1 where `substring` cannot be found. |
```